### PR TITLE
Closes #131

### LINKED ISSUE #131: New helper for bivariate normal parameter validation
Bivariate Normal distribution appears quite often in many crmPack models. It would be good to have some dedicated function for validation of bivariate normal parameters, i.e. mean, cov/prec, so that (bivariate normal based) model validations functions can easily use it.

Something like below. The open issue is that whether functions should return validate class object (i.e. `o` in the example below), or just result of validation (`o$result()`). 

```
h_test_normal_params <- function(params) {
  o <- Validate()

  o$check(
    test_list(params, len = 3, names = c("mean", "cov", "prec")),
    "must be a list with elements mean, cov, prec",
  )
  o$check(
    test_numeric(x = params$mean, len = 2L, any.missing = FALSE),
    "mean must have length 2 and no missing values are allowed"
  )
  is_cov_2x2 <- test_matrix(
    params$cov,
    mode = "numeric", nrows = 2, ncols = 2, any.missing = FALSE
  )
  is_prec_2x2 <- test_matrix(
    params$prec,
    mode = "numeric", nrows = 2, ncols = 2, any.missing = FALSE
  )
  o$check(is_cov_2x2, "cov must be 2x2 matrix without any missing values")
  o$check(is_prec_2x2, "prec must be 2x2 matrix without any missing values")
  if (is_cov_2x2) {
    o$check(
      h_is_positive_definite(params$cov),
      "cov must be positive-definite matrix"
    )
    if (is_prec_2x2) {
      o$check(
        all.equal(params$cov %*% params$prec, diag(1, 2), check.attributes = FALSE) == TRUE,
        "prec must be inverse of cov"
      )
    }
  }
  o$result()
}
```

### PR DESCRIPTION
**Pull request by wwojciech**
Closes #131 (no unit tests yet).

### CODE REVIEW

File: R/helpers.R [hunk 1]
```diff
@@ -42,6 +42,31 @@ Validate <- setRefClass(
...
+#' `character` messages supplied to the input of this function into one `character`
+#' vector, or simply returns `TRUE` if there were no such messages.
+#'
+#' @param ... outputs from S4 object validation functions.
+#'
+#' @return `character` vector if there was at least object of type `character`
+#'   in the input. Otherwise, `TRUE`.
+#'
+#' @export
+h_combine_validation_results <- function(...) {
```
Comment (danielinteractive) [inline R/helpers.R:60]: I thought we had already a helper for this? or not?

**Review by danielinteractive — COMMENTED** [review]
thx Wojciech, good idea, pls proceed

File: R/helpers.R (hunk 1 above)
Comment (wwojciech) [reply R/helpers.R:60]: Nope, this is a new thing.

File: R/helpers.R [hunk 2]
```diff
@@ -42,33 +42,36 @@ Validate <- setRefClass(
...
+# #'
+# #' @return `character` vector if there was at least object of type `character`
+# #'   in the input. Otherwise, `TRUE`.
+# #'
+# #' @export
+# h_combine_validation_results <- function(...) {
+#   x <- list(...)
+#   assert_true(all(sapply(x, function(i) is.character(i) || isTRUE(i))))
+#
+#   is_msg <- sapply(x, is.character)
```
Comment (danielinteractive) [inline R/helpers.R:67]: please delete the code when not needed

File: tests/testthat/test-helpers.R [hunk 3]
```diff
@@ -1,3 +1,44 @@
+# h_combine_validation_results ----
+
+# nolint start
+
+# test_that("h_combine_validation_results returns TRUE for all TRUE objects", {
```
Comment (danielinteractive) [inline tests/testthat/test-helpers.R:5]: please delete when not needed

**Review by danielinteractive — APPROVED** [review]
thx Wojciech, nice!

**Comment by wwojciech** [comment]
Tx @danielinteractive I will just check if all vignettes are rendering correctly and then will merge it.

**Pull request merged**